Purchases/LocalReceiptParsing/DataConverters/ArraySlice_UInt8+Extensions.swift

```diff
@@ -44,7 +44,8 @@ extension ArraySlice where Element == UInt8 {
 
     func toDate() -> Date? {
         guard let dateString = String(bytes: Array(self), encoding: .ascii) else { return nil }
-        return DateFormatter.date(fromISO8601SecondsOrMillisecondsString: dateString)
+
+        return ISO8601DateFormatter.default.date(from: dateString)
```

Comment:
[totally minor] would it make sense to create a static method that uses the default formatter to keep the same structure?

Reply:
What do you mean by a static method? `ISO8601DateFormatter.default()` instead?

Reply:
Not exactly. What I mean is something like this:

```swift
ISO8601DateFormatter.date(from: dateString)
```

Reply:
I see. I'm not against it, but it can be confusing to have 2 ways of doing the same thing. It always leads to the question: "what's the difference?". Having only one API solves that problem.

Reply:
Yes, it makes sense to keep it simple.  👍🏼